package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "moduleconfig": "^2.0.0",
```

Comment:
This will be updated after releasing changes from https://github.com/ebdrup/moduleconfig/pull/6